analyzer/consensus/consensus.go

```diff
@@ -721,13 +721,15 @@ func (m *Main) queueEscrows(batch *storage.QueryBatch, data *storage.StakingData
 				UPDATE %s.accounts
 					SET
 						escrow_balance_active = escrow_balance_active - $2,
-						escrow_balance_debonding = escrow_balance_debonding + $3,
-						escrow_total_shares_debonding = escrow_total_shares_debonding + $2
+						escrow_total_shares_active = escrow_total_shares_active - $3,
+						escrow_balance_debonding = escrow_balance_debonding + $2,
+						escrow_total_shares_debonding = escrow_total_shares_debonding + $4
 					WHERE address = $1;
 			`, chainID),
 				e.DebondingStart.Escrow.String(),
-				e.DebondingStart.ActiveShares.ToBigInt().Uint64(),
 				e.DebondingStart.Amount.ToBigInt().Uint64(),
+				e.DebondingStart.ActiveShares.ToBigInt().Uint64(),
+				e.DebondingStart.DebondingShares.ToBigInt().Uint64(),
```

Comment:
To confirm this:

- Subtracts `ActiveShares` from `escrow_total_shares_active`
- Adds `Amount` to `escrow_balance_debonding`
- Adds `DebondingShares` to `escrow_total_shares_debonding`

Reply:
The only common is that `Amount` is also subtracted from `escrow_balance_active`.